config.go

```diff
@@ -154,12 +155,33 @@ func ConfigStringToClusterConfig(configString string) (*gocql.ClusterConfig, err
 				case "password":
 					passwordAuthenticator.Password = value
 					clusterConfig.Authenticator = passwordAuthenticator
+				case "enableHostVerification":
+					if sslOptions == nil {
+						sslOptions = &gocql.SslOptions{}
+					}
+					sslOptions.EnableHostVerification = (value == "true")
```

Comment:
What do you think about using https://golang.org/pkg/strconv/#ParseBool ?

Reply:
oops, will do